README.md

```diff
@@ -74,3 +74,42 @@ To update CKEditor in liferay-portal:
 3. Re-deploy the module with `gradlew clean deploy`.
 
 An example of this can be seen in [this](https://github.com/liferay/liferay-portal/commit/5b2ae3732d96f7f0dec6d35cb4de99f9d389c248) commit (look at the [`package.json`](https://github.com/liferay/liferay-portal/blob/5b2ae3732d96f7f0dec6d35cb4de99f9d389c248/modules/apps/frontend-editor/frontend-editor-ckeditor-web/package.json) file)
+
+## Patching
+
+1. Make sure you're update to date with the [superproject](https://github.com/liferay/liferay-ckeditor) repository:
+
+	```sh
+	git pull origin master
+	```
+
+2. Set up everything to start working on a patch:
+
+	```sh
+	sh ck.sh setup
+	```
+
+3. Work on your changes:
+
+	This could be `cherry-pick`ing a previously created commit or manually editing a file, so this can't be automated.
+
+4. Create your commit, add your changes and write a good commit message.
+
+5. Navigate back to the superproject's root directory and create the patch:
+
+	```sh
+	cd ..
+	sh ck.sh patch
+	```
+
+7. Create a build of CKEditor containing the patches:
```

Comment:
In the rendered output, this is "6". You could update this (and the following one) to match, or leave it. Depends on whether you care more about [minimizing effort](http://wiki.c2.com/?LazinessImpatienceHubris) or making the source Markdown as readable/correct as possible. For example, some people always just make every numbered list item "1." in Markdown.

Reply:
That's interesting ... good point
As for the "fix" , I habe my doubts, I can always update it since this is easy enought, but it might be better to use bullet points? For example, in two months we might discover that there is a missing step and we'll have to "update" everything again.

Reply:
Right, that's exactly why some people write numbered lists like this in Markdown:

```markdown
1. first
1. second
1. third
```

In [the words](https://daringfireball.net/projects/markdown/syntax) of the immortal(?) John Gruber himself:

> The point is, if you want to, you can use ordinal numbers in your ordered Markdown lists, so that the numbers in your source match the numbers in your published HTML. But if you want to be lazy, you don’t have to.

I don't really know what the right answer is. In general, I like being lazy, but I always felt weird numbering things as "1.", "1.", "1." etc so I never did it.